ConceptMap/ConceptMap-Genomics-specimen-primarysample.json

```diff
@@ -0,0 +1,309 @@
+        {
+          "code": "chorionic-villus-sampling",
+          "display": "Chorionic Villus Sampling",
+          "target": [
+            {
+              "code": "258565009",
+              "display": "Chorionic villi specimen",
+              "equivalence": "equivalent"
+            }
+          ]
+        },
+        {
+          "code": "fetal-blood",
+          "display": "Fetal Blood",
+          "target": [
+            {
+              "code": "737357006",
+              "display": "Fetal blood specimen",
+              "equivalence": "equivalent"
+            }
+          ]
+        },
+        {
+          "code": "fibroblasts",
+          "display": "Fibroblasts",
+          "target": [
+            {
+              "code": "119333000",
+              "display": "Fibroblast specimen",
+              "equivalence": "equivalent"
+            }
+          ]
+        },
+        {
+          "code": "heart",
+          "display": "Heart",
+          "target": [
+            {
+              "code": "127462005",
+              "display": "Specimen from heart (specimen)",
+              "equivalence": "equivalent"
+            }
+          ]
+        },
+        {
+          "code": "kidney",
+          "display": "Kidney",
+          "target": [
+            {
+              "code": "127473003",
+              "display": "Specimen from kidney (specimen)",
+              "equivalence": "equivalent"
+            }
+          ]
+        },
+        {
+          "code": "liver",
+          "display": "Liver",
+          "target": [
+            {
+              "code": "119383005",
+              "display": "Specimen from liver (specimen)",
+              "equivalence": "equivalent"
+            }
+          ]
+        },
+        {
+          "code": "lymph-node",
+          "display": "Lymph Node",
+          "target": [
+            {
+              "code": "258589002",
+              "display": "Lymph node specimen",
+              "equivalence": "equivalent"
+            }
+          ]
+        },
+        {
+          "code": "other",
+          "display": "Other",
+          "target": [
+            {
+              "code": "123038009",
+              "display": "Specimen (specimen)",
+              "equivalence": "wider",
+              "comment": "Any other suitable descendant of 123038009|Specimen (specimen) not listed on this code. This should be provided by free text."
+            }
+          ]
+        },
+        {
+          "code": "pericardial",
+          "display": "Pericardial",
+          "target": [
+            {
+              "code": "122571007",
+              "display": "Pericardial fluid specimen (specimen)",
+              "equivalence": "equivalent"
+            }
+          ]
+        },
+        {
+          "code": "peritoneal",
+          "display": "Peritoneal",
+          "target": [
+            {
+              "code": "122571007",
+              "display": "Pericardial fluid specimen (specimen)",
```

Comment:
Maybe 430250001 | Specimen from peritoneum (specimen) |, though there is a fluid sample which does not come under this tree, may need clarification as to whether this is a tissue or fluid specimen